apps/microsoft-teams/frontend/src/components/config/EventsSelection/EventsSelection.tsx

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from 'react';
+import { Box, Checkbox, FormControl } from '@contentful/f36-components';
+import { Notification } from '@customTypes/configPage';
+import { AppEventKey, eventsSelection } from '@constants/configCopy';
+
+interface Props {
+  notification: Notification;
+  handleNotificationEdit: (notificationEdit: Partial<Notification>) => void;
+}
+
+const EventsSelection = (props: Props) => {
+  const { notification, handleNotificationEdit } = props;
+
+  // For Checkbox.Group, creates an array of the values of the checkboxes that should be checked
+  const selectedEventsArray = useMemo(
+    () =>
+      Object.values(AppEventKey).reduce((acc: string[], event) => {
```

Comment:
Cool, small thought, could this not be more simply implemented as a filter? 🤔 

```ts
Object.values(AppEventKey).filter(event => notification.selectedEvents[event])
```

Reply:
Yes, good call!